packages/db/schema.ts

```diff
@@ -157,8 +157,11 @@ export const bookmarkAssets = sqliteTable("bookmarkAssets", {
     .primaryKey()
     .$defaultFn(() => createId())
     .references(() => bookmarks.id, { onDelete: "cascade" }),
-  assetType: text("assetType", { enum: ["image"] }).notNull(),
+  assetType: text("assetType", { enum: ["image", "pdf"] }).notNull(),
   assetId: text("assetId").notNull(),
+  content: text("content"),
+  metadata: text("metadata"),
+  info: text("info"),
```

Comment:
What exactly gets added in the `info` and `metadata`? Are they really worth storing and indexing?

Reply:
In case of a PDF, the metadata will have any PDF meta that are added to the PDF file like the author and so on. 
For the info, it's any relevant information about the PDF version. 

Honestly I kept them for the sake of applying the same for the images as well.
Basically, we are sending the image to openai, why not ask openai to provide both the tags and a description about the image ?
The cost is the same and it will enhance the images search by a huge amount with no extra cost.

Reply:
My main concern about them is the fact that they are not structured data and I'm not a big fan of dumping schema-less jsons into the database. Basically, they'll never be useful beyond getting indexed for search. But if you think they provide useful data, then why not. I think we can keep `metadata` and drop `info`? Or merge info into `metadata`?

> why not ask openai to provide both the tags and a description about the image ?
That I 100% agree with and this can be in the `content` field in the database. My comment is in particular about `info` and `metadata`.

Reply:
I agree.
I'll dump the info and keep the metadata.